synapse/lib/agenda.py

```diff
@@ -886,6 +891,7 @@ async def _runJob(self, user, appt):
                 success = True
                 result = f'finished successfully with {count} nodes'
             finally:
+                await self.core.feedBeholder('cron:stop', {'iden': appt.iden})
```

Comment:
is this safe to do if we've been fini'd?

Reply:
Probably not. I'll add some logic.